cmd/snap-bootstrap/bootstrap/bootstrap.go

```diff
@@ -92,6 +93,15 @@ func Run(gadgetRoot, device string, options Options) error {
 	if err := diskLayout.RemoveCreated(); err != nil {
 		return fmt.Errorf("cannot remove partitions from previous install: %v", err)
 	}
+	// at this point we removed any existing partition, nuke any of the
+	// existing keyfiles too (LP: #1879338)
```

Comment:
// existing sealed key files placed outside of the encrypted partitions (LP: #1879338)